File: src/main/java/eu/mshade/enderframe/entity/LivingEntity.java
package eu.mshade.enderframe.entity;

import eu.mshade.enderframe.world.Location;
import eu.mshade.enderframe.world.Vector;

import java.util.UUID;

public abstract class LivingEntity extends Entity {

    /*
    protected float health;
    protected int potionEffectColor;
    protected boolean potionEffectAmbient;
    protected int numberOfArrowInEntity;
    protected boolean ai;

     */


    public LivingEntity(Location location, Vector velocity, int entityId, UUID uuid, EntityType entityType) {
        super(location, velocity, entityId, uuid, entityType);
    }

    public LivingEntity(Location location, int entityId, EntityType entityType) {
        super(location, entityId, entityType);
    }


}

File: src/main/java/eu/mshade/enderframe/entity/LivingEntity.kt
package eu.mshade.enderframe.entity

import eu.mshade.enderframe.world.Location
import eu.mshade.enderframe.world.Vector
import java.util.*

abstract class LivingEntity : Entity {

    constructor(location: Location, velocity: Vector, entityId: Int, uuid: UUID, entityKey: EntityKey) : super(
        location,
        velocity,
        entityId,
        uuid,
        entityKey
    )

    constructor(location: Location, entityId: Int, entityKey: EntityKey) : super(location, entityId, entityKey)
}